### PR TITLE
feat: add contact page

### PR DESCRIPTION
**Pull request by Rgascoin**
# Description
Add Contact Page and cypress setup + contact's page test

# Changes include
[ ] New feature (non-breaking change that adds functionality)
[ ] Bugfix (non-breaking change that solves an issue)
[x] Breaking change (change that is not backwards-compatible and/or changes current functionality)
# Checklist
...

### CODE REVIEW

File: cypress/integration/Contact/contact_form.spec.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    cy.contains('Contactez Datadvisor')
+    cy.url().should('include', '/contact')
+  })
+
+  it('Check All errors', () => {
+    cy.visit('http://localhost:3000/contact')
+    cy.contains('Envoyer').click()
+
+    cy.contains('Le prénom ne peux être vide')
```
Comment (MrZalTy) [inline cypress/integration/Contact/contact_form.spec.js:15]: Fix misspelling

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,12686 @@
+{
```
Comment (MrZalTy) [inline package-lock.json:1]: Remove this file

**Review by MrZalTy — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -48,6 +53,8 @@
     "@headlessui/react": "^1.5.0",
     "@heroicons/react": "^1.0.6",
     "@tailwindcss/aspect-ratio": "^0.4.0",
+    "@tailwindcss/forms": "^0.5.0",
+    "cypress": "^9.6.0",
```
Comment (MrZalTy) [inline package.json:57]: cypress, @tailwindcss/forms, @tailwindcss/aspect-ratio should be in devDependencies.

**Review by MrZalTy — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (Rgascoin) [reply package.json]: Fixed in [0bf8ee0](https://github.com/Datadvisor/frontend/pull/3/commits/0bf8ee0eec754cc288091916fbef0178ccf5ccb2)

File: package-lock.json (hunk 2 above)
Comment (Rgascoin) [reply package-lock.json]: Fixed in [992bcb1](https://github.com/Datadvisor/frontend/pull/3/commits/992bcb145015d28d7d7e5adc7c7e4a8f293431e8)

File: cypress/integration/Contact/contact_form.spec.js (hunk 1 above)
Comment (Rgascoin) [reply cypress/integration/Contact/contact_form.spec.js]: Fixed in [d88c4b2](https://github.com/Datadvisor/frontend/pull/3/commits/d88c4b250dd8ac626e24bac6b84e487b33e01db6)

**Review by MrZalTy — APPROVED** [review]
OK

**Pull request merged**